src/main/java/mekanism/common/util/InventoryUtils.java

```diff
@@ -70,12 +71,21 @@ public static TransitResponse putStackInInventory(TileEntity tile, TransitReques
         return TransitResponse.EMPTY;
     }
 
+    /**
+     * Like {@link ItemHandlerHelper#canItemStacksStack(ItemStack, ItemStack)} but empty stacks mean equal (either param).
+     * Thiakil: not sure why.
+     *
+     * @param toInsert stack a
+     * @param inSlot stack b
+     * @return true if they are compatible
+     */
     public static boolean areItemsStackable(ItemStack toInsert, ItemStack inSlot) {
```

Comment:
This can (or more accurately `!areItemsStackable()`) probably now be used for https://github.com/mekanism/Mekanism/blob/1.12/src/main/java/mekanism/common/tile/TileEntityFactory.java#L412